sync/sync-impl/src/main/java/com/duckduckgo/sync/impl/auth/SupportedDeviceAuthChecker.kt

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright (c) 2023 DuckDuckGo
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.duckduckgo.sync.impl.auth
+
+import android.app.KeyguardManager
+import android.content.Context
+import androidx.biometric.BiometricManager
+import androidx.biometric.BiometricManager.Authenticators
+import androidx.biometric.BiometricManager.BIOMETRIC_SUCCESS
+import com.duckduckgo.di.scopes.AppScope
+import com.squareup.anvil.annotations.ContributesBinding
+import javax.inject.Inject
+
+interface SupportedDeviceAuthChecker {
```

Comment:
This is used from `hasValidDeviceAuthentication`,  which seems unused. do we tell the user that needs to have auth enabled?

Reply:
This is no longer a problem. If the user doesn’t have auth enabled we will just go ahead for now.